src/main/java/hudson/plugins/analysis/collector/AnalysisPublisher.java

```diff
@@ -149,6 +68,21 @@ public boolean isCheckStyleActivated() {
     }
 
     /**
+     * Needed by workflow snippet generator.
+     * Having this ugly getter naming to keep backward configuration file compatibility on fields names.
+     * 
+     * @see #isCheckStyleActivated()
+     */
+    public boolean isIsCheckStyleActivated() {
```

Comment:
Would stapler work if you just leave <code> isCheckStyleActivated()</code>? According from the presentation from @daniel-beck it should work, but I'm not sure.